src/app/enums/path.enum.ts

```diff
@@ -1,8 +1,8 @@
-export enum Path{
-    dashboard = "#/dashboard",
-    home = "#",
-    profile = "#/userId/profile",
-    resetPassword = "#/reset-password",
-    signIn = "#/sign-in",
-    signUp = "#/sign-up"
-};
+export enum Path {
+  Dashboard = '#/dashboard',
+  Home = '',
+  Profile = '/profile',
```

Comment:
Please, check about `#` here

Reply:
not need, because try added user ID before "/profile"